client/src/app/project-form/project-form.component.html

```diff
@@ -105,22 +105,31 @@
               <label for="manager" class="control-label"
                 >Project Manager:</label
               >
+             
               <mat-select
+             
                 class="browser-default custom-select"
                 name="manager"
                 [ngModel]="(project && project?.projectManager)"
                 class="form-control"
                 [disabled]="formType === 'get'"
                 (ngModelChange)="addProjectManager($event)"
                 #projectManagerInput="ngModel"
-                required
-              >
-
-                <ng-template ngFor let-emp [ngForOf]="empList">
-                  <mat-option *ngIf="!projMembers.includes(emp._id)" [value]="emp._id">{{ emp.empId }}----{{
-                    emp.name
-                  }}</mat-option>
-                </ng-template>
+                required>
+                <i class="fa fa-search "></i>
+                <input id="typeahead-config" type="text" class="my-control" [(ngModel)]="(project?.projectManager)[0]" 
```

Comment:
please include typeahead in the refactored version of the code

Reply:
it is added